source/documentation/guides/baseline_coding_standards.html.md.erb

```diff
@@ -0,0 +1,36 @@
+---
+title: Baseline Coding Standards
+last_reviewed_on: 2021-04-27
+review_in: 6 months
+owner_slack: '#opg-developers'
+---
+
+# <%= current_page.data.title %>
+
+Baselines coding standards for OPG that teams can customise and adapt to their needs, starting from a common point
+
+## Standards
+
+- Code should conform to a recognised style guides or standards, and take advantage of automatic style checkers, linters, and static analysis tools
+- Project structures should follow the conventions of the language and frameworks used
+- Favour clear variable and function names over acronyms, abbreviations and explanations in comments (and don’t assume technical and business domain knowledge)
+- Commented where it aids understanding and good naming cannot solve this
+- Code should be readable
+- Code should be concise, but not at the expense of readability
+- Code should be tested, automatically and with easily visible measures of what is covered
+- Favour open standards over proprietary or customised ones
+- Code and design should be secure, and make use of automated tools like dependency scanners
+- Maintained; addressing emerging user feedback, with libraries and kept up to date, and agreed pattern changes applied consistently across the application
+- The state of the code should be observable, with adequate logging, tracing, metrics and alerts. To understand the state of the system, be aware of issues, and see the impact of changes
+- Appropriately performant and scalable
+- Any information or user interact-able parts of the system must be Accessible
```

Comment:
I don't think interactable needs a hyphen

Reply:
Apparently the words existence is a subject of much debate, but if it does exist then yep it doesn't need a hyphen 😄